assignments/arrays.js

```diff
@@ -63,34 +63,69 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+
+console.log(inventory[32]);
 
 
 
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
-let lastCar = 0;
-console.log();
+let lastCar = (inventory.length)-1;
+console.log(inventory[lastCar]);
 
 // ==== Challenge 3 ====
 // The marketing team wants the car models listed alphabetically on the website. Sort all the car model names into alphabetical order and log the results in the console
-let carModels = [];
-console.log();
+let carModels=[]
+for (i=0; i<inventory.length; i++) {
+    
+    carModels.push((inventory[i]['car_model']));
```

Comment:
You could also have used dot notation here. Sometimes it can be a little more visually clear. Ex:
```carModels.push(inventory[i].car_model);```
Serves the exact same purpose and if brackets work for you then use them. I just wanted you to be aware of the different options. Dot notation is more popular.